src/modules/translation/languages/en.json

```diff
@@ -173,6 +173,16 @@
     "title": "Upload Scene",
     "upload_manually": "Upload Manually"
   },
+  "quota_exceeded_modal": {
+    "sign_in": {
+      "title": "Oh noes!",
+      "description": "You ran out of anonymous storage space, please sign in to continue"
```

Comment:
Maybe `browser's storage` instead of `anonymous storage` to keep it consistent with the `  toast.localStorage` message?

Also this is missing translations for the other languages